Userland/Applications/Spreadsheet/SpreadsheetView.cpp

```diff
@@ -297,7 +297,9 @@ SpreadsheetView::SpreadsheetView(Sheet& sheet)
         }
 
         if (event.mime_data().has_text()) {
-            auto* target_cell = m_sheet->at({ (size_t)index.column(), (size_t)index.row() });
+            const Position target { (size_t)index.column(), (size_t)index.row() };
+            m_sheet->ensure(target);
+            auto* target_cell = m_sheet->at(target);
```

Comment:
`Sheet::ensure()` returns the cell IIRC,

```c++
auto* target_cell = &m_sheet->ensure(target);
```
to avoid a double-lookup

Reply:
Yes, that is correct and I tried to use that but did not get it to compile. I now realized that I forgot the `&` after `auto` hehe. 
Pushing a fix using the return value :+1: